build_dirs/build_docs.py

```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import json
+from pathlib import Path
+
+
+BASE = Path(os.path.abspath(os.path.dirname(__file__)))
```

Comment:
Just check before changing, but I think that
`Path(os.path.abspath(os.path.dirname(__file__)))` == `Path(__file__).parent`